components/dashicon/index.js

```diff
@@ -343,6 +351,9 @@ export default class Dashicon extends wp.element.Component {
 			case 'editor-ltr':
 				path = 'M5.52 2h7.43c.55 0 1 .45 1 1s-.45 1-1 1h-1v13c0 .55-.45 1-1 1s-1-.45-1-1V5c0-.55-.45-1-1-1s-1 .45-1 1v12c0 .55-.45 1-1 1s-1-.45-1-1v-5.96h-.43C3.02 11.04 1 9.02 1 6.52S3.02 2 5.52 2zM14 14l5-4-5-4v8z';
 				break;
+			case 'editor-ol-rtl':
```

Comment:
I assume this and "instagram" are expected additions?

Reply:
Yes, they were added to the upstream Dashicons sprite set as part of separate efforts.